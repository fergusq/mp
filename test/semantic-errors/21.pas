program p;
procedure f();
begin writeln(1)end;
begin f():=2;
end.
