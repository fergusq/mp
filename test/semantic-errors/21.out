Semantic error: Left side of assignment is not lval
